README.md

```diff
@@ -78,11 +78,10 @@ export LIBVA_DRIVER_NAME=iHD
 * BDW (Broadwell)
 * SKL (Skylake)
 * BXT (Broxton) / APL (Apollo Lake)
-* KBL (Kaby Lake)
-* CFL (Coffee Lake)
-* WHL (Whiskey Lake)
-* CML (Comet Lake)
+* KBL* (KBL/Kaby Lake; CFL/Coffe Lake; WHL/Whiskey Lake; CML/Comet Lake; AML/Amber Lake)
 * ICL (Ice Lake)
+* JSL (Jasper Lake)/EHL (Elkhart Lake)
```

Comment:
Don't JSL/EHL belong to gen11 family? Maybe you need to list them in ICL line? /this gives issue for the table below/